db/version_edit.h

```diff
@@ -119,8 +121,9 @@ class VersionEdit {
   void DeleteFile(int level, uint64_t file) {
     deleted_files_.insert(std::make_pair(level, file));
   }
+  size_t DeletedFileCount() const {return(deleted_files_.size());};
 
-  void EncodeTo(std::string* dst, bool format2=false) const;
+  void EncodeTo(std::string* dst, bool format2=true) const;
```

Comment:
Anticipating the possible need in the future to extend the serialized format, it might be a good idea to change the format indicator from a bool to an enumeration.